### PR TITLE
(core): normalize remaining module names

### PR DESCRIPTION
**Pull request by icfantv**

### CODE REVIEW

**Comment by icfantv** [comment]
@anotherchrisberry @zanthrash PTAL

File: app/scripts/modules/core/help/helpContents.registry.spec.ts [hunk 1]
```diff
@@ -1,12 +1,11 @@
-import {mock} from 'angular';
-import helpRegistryModule, {HelpContentsRegistry} from './helpContents.registry';
+  import {mock} from 'angular';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/help/helpContents.registry.spec.ts]: probably don't want this space

File: app/scripts/modules/core/modal/wizard/v2modalWizard.component.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import {module} from 'angular';
-import wizardPageModule from './v2wizardPage.component';
-import modalWizardServiceModule from './v2modalWizard.service';
+import {V2_WIZARD_PAGE_COMPONENT} from './v2wizardPage.component';
+import {V2_MODAL_WIZARD_SERVICE} from 'core/modal/wizard/v2modalWizard.service';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/modal/wizard/v2modalWizard.component.ts]: leave path relative, i.e. `./`

File: app/scripts/modules/core/modal/wizard/v2wizardPage.component.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import {module} from 'angular';
 
-import modalWizardServiceModule from './v2modalWizard.service';
+import {V2_MODAL_WIZARD_SERVICE} from 'core/modal/wizard/v2modalWizard.service';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/modal/wizard/v2wizardPage.component.ts]: relative path is fine here

File: app/scripts/modules/core/modal/wizard/wizardSubFormValidation.service.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 import _ from 'lodash';
-import modalWizardServiceModule from './v2modalWizard.service';
+import {V2_MODAL_WIZARD_SERVICE} from 'core/modal/wizard/v2modalWizard.service';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/modal/wizard/wizardSubFormValidation.service.js]: relative path

File: app/scripts/modules/core/modal/wizard/wizardSubFormValidation.service.spec.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import modalWizardServiceModule from './v2modalWizard.service';
+import {V2_MODAL_WIZARD_SERVICE} from 'core/modal/wizard/v2modalWizard.service';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/modal/wizard/wizardSubFormValidation.service.spec.js]: relative path

**Pull request merged**